File: 2_samana/EIQC/01_ejercicio_Sem2/seed.py
from models import db, Producto

def seed_data(app):
    with app.app_context():
        db.drop_all()
        db.create_all()
        productos = [
            Producto(nombre="Laptop", descripcion="Laptop de última generación", precio=1500.00),
            Producto(nombre="Mouse", descripcion="Mouse ergonómico", precio=25.50),
            Producto(nombre="Teclado", descripcion="Teclado mecánico", precio=75.00),
        ]
        db.session.add_all(productos)
        db.session.commit()
        print("Base de datos inicializada con datos de prueba.")

File: 2_samana/MAAAA-Flask_crud_app/crud_js_react/backend/seed.js
// backend/seed.js
const knex = require('./db');

async function seedDatabase() {
  await knex('productos').del(); // Eliminar datos previos
  await knex('productos').insert([
    { nombre: 'Laptop', descripcion: 'Laptop de última generación', precio: 1200.99 },
    { nombre: 'Teléfono', descripcion: 'Smartphone con cámara avanzada', precio: 800.50 },
    { nombre: 'Monitor', descripcion: 'Pantalla LED 24 pulgadas', precio: 199.99 },
  ]);
  console.log('Base de datos inicializada con datos de prueba');
  process.exit(); // Salir del script después de insertar
}

seedDatabase().catch((err) => console.error(err));

